js/src/bugsnag-cli-wrapper.ts

```diff
@@ -25,7 +25,7 @@ class BugsnagCLI {
                     if (typeof value === 'boolean' && value === true) {
                         return [`--${kebabKey}`]
                     } else if (typeof value !== 'boolean') {
-                        return [`--${kebabKey}`, String(value)]
+                        return [`--${kebabKey}=${String(value)}`]
```

Comment:
Can we get a test in to cover this? ... and any other more complex cases you can think of for metadata fields?